src/Controls/src/Core/Layout/GridLayoutExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Microsoft.Maui.Controls
+{
+	public static class GridLayoutExtensions 
+	{
+		public static void Add(this GridLayout gridLayout, IView view, int column = 0, int row = 0)
+		{
+			if (view == null)
+				throw new ArgumentNullException(nameof(view));
+			if (column < 0)
+				throw new ArgumentOutOfRangeException(nameof(column));
+			if (row < 0)
+				throw new ArgumentOutOfRangeException(nameof(row));
+
+			gridLayout.Add(view);
```

Comment:
I have been wanting to ask about this, is there a way to add a child directly to the required row and column? I don't know what happens when you call `.Add` but I'm assuming it triggers a layout in the default row and column assigned by `.Add` and then the 2 `.Set`s below trigger another 2 layouts. Might be a bit of a performance hit, right?